understanding/target-size.html

```diff
@@ -203,10 +203,11 @@ <h2>重要な用語</h2>
             <dd><definition xmlns="">
                   					
                   
-                  <p xmlns="http://www.w3.org/1999/xhtml">マウス、ペン、タッチ接触のように、画面上の特定の (1 つ又は複数の) 地点を指すことができる入力デバイス。</p>
+                  <p xmlns="http://www.w3.org/1999/xhtml">マウス、ペン、タッチ接触のように、画面上の特定の (1 つ又は複数の) 地点を指すことができるデバイスからの入力。</p>
```

Comment:
coordinateがどこかに行ってしまっているのと、ここは実は（達成基準の）ターゲットでよいのでは…？

```suggestion
                  <p xmlns="http://www.w3.org/1999/xhtml">マウス、ペン、タッチ接触のように、画面上の特定の (1 つ又は複数の) 座標をターゲットにできるデバイスからの入力。</p>
```